code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -149,7 +149,6 @@
 	name = ".357 heartseeker bullet"
 	icon_state = "gauss"
 	damage = 50
-	homing = TRUE
```

Comment:
what does this var do if it doesn't home?

Reply:
this var just changes how the projectile processes movement.
funnily enough -
normally it's only set up here, when a target is passed in.
https://github.com/tgstation/tgstation/blob/a46523adb68553a49be819eabce6966bc130eb29/code/modules/projectiles/projectile.dm#L811-L822

in the movement processing code, homing is checked here, and if it's true then it changes how it handles movement, but otherwise it does not in and of itself cause homing
https://github.com/tgstation/tgstation/blob/a46523adb68553a49be819eabce6966bc130eb29/code/modules/projectiles/projectile.dm#L1047-L1050

presumably it was supposed to home in on the nearest valid target passively, something like how magic missile works (iirc), but i guess it stopped working or never actually worked?

Reply:
Fucking agony

Reply:
maybe set it to `VAR_FINAL` in that case

Reply:
It never worked yeah, I didn't change that bit of projectile logic. Homing always required set_homing_target